Game/Assets/Scripts/src/Script/MenuController.cpp

```diff
@@ -0,0 +1,31 @@
+#include "MenuController.h"
+
+#include "Main/Application.h"
+#include "Module/ModuleInput.h"
+
+
+bool MenuController::ConfirmMovedUp( ModuleInput& input)
```

Comment:
Why are u passing the ModuleInput by parameter? And not doing App->input-> directly?

Reply:
Because only Script classes have access to it, and here I have a static class that is not a script.